src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -45,12 +47,22 @@ public void storeMessage(Message message) {
     datastore.put(messageEntity);
   }
 
-    /** Stores FoodItem in Datastore. */
+    /** Stores FoodItem in Datastore.
+      * Prevent the same food from being entered into the database more than once
+      */
+
   public void storeFood(FoodItem food) {
-    Entity foodEntity = new Entity("FoodItem", food.getID().toString());
-    foodEntity.setProperty("Name", food.getName());
-    foodEntity.setProperty("CO2", food.getCO2PerYear());
-    datastore.put(foodEntity);
+    Set<String> existingfood = this.getFoodItem();
```

Comment:
It might be worth while to create a function like
```
public boolean exists(FoodItem food) {
  Set<String> existingFoods = this.getAllFoodItems();
  Set<String> lowercaseFoods;
  for (String food : existingFoods) {
    lowercaseFoods.add(food.toLowerCase());
  }
  return lowercaseFoods.contains(food.getName().toLowerCase());
}
```